website/docs/error-reference.md

```diff
@@ -702,6 +702,68 @@ you'll get this error—the latter export is redundant, as it's implied by expor
 
 To fix this error, simply remove the more specific export.
 
+## 3717
+
+> This error is specific to Stripe's custom `--stripe-packages` definition. If
+> you are at Stripe, please see [go/modularity](http://go/modularity) for more.
+
+This error means that you're using a name that's private in another package, and
+not epxorted.
+
+To fix this error, you'll need to either export the symbol in the other package,
+or figure out a way around using it.
+
+## 3718
+
+> This error is specific to Stripe's custom `--stripe-packages` definition. If
+> you are at Stripe, please see [go/modularity](http://go/modularity) for more.
+
+This error means that you're using a symbol from a packge that's not imported.
+
+You can fix the error by adding an `import` statement for the package that's
+mentioned to your package's `__package.rb` file.
+
+## 3719
+
+> This error is specific to Stripe's custom `--stripe-packages` definition. If
+> you are at Stripe, please see [go/modularity](http://go/modularity) for more.
+
+This error means that you're using a symbol defined in a package from an
+unpackaged context.
+
+To fix this error you'll need to figure out a way around using that symbol, as
+sorbet does not allow packaged code to be used from an unpackaged context.
```

Comment:
Can we make this example more concrete or provide more details? It seems like this wording is just restating the error header

For example, something which i'm not sure is true or not, but gets at the style of the edit I'm hoping for:

```
Files that are marked `# packaged: false` must only define and reference
completely unpackaged code. For example, # packaged: false files can be used
to monkeypatch classes defined by the standard library. However, these monkey
patches cannot mention packaged code. To monkeypatch something in the standard
library and also mention a packaged definition do X.
```

Reply:
I like the suggestion about mentioning `packaged: false` here. I'll take a pass at rewording this one.